src/Sentry.Unity.Editor/SentryWindow.cs

```diff
@@ -38,7 +50,28 @@ private void OnEnable()
                 AssetDatabase.CreateAsset(Options, SentryOptionsAssetPath);
             }
 
-            EditorUtility.SetDirty(Options);
+            EditorUtility.SetDirty(Options);*/
+        }
+
+        private UnitySentryOptionsJson LoadSentryJsonConfig()
+        {
+            if (!File.Exists(SentryOptionsJsonPathFull))
+            {
+                var unitySentryOptionsJson = new UnitySentryOptionsJson
+                {
+                    Dsn = "https://94677106febe46b88b9b9ae5efd18a00@o447951.ingest.sentry.io/5439417",
+                    Enabled = true
+                };
+                var emptyOptions = JsonSerializer.Serialize(unitySentryOptionsJson, _jsonOptions);
+                File.WriteAllText(SentryOptionsJsonPathFull, emptyOptions);

```

Comment:
Isn't there a `Stream` API we can use instead of getting a `string` allocated?

Reply:
Streams are only in `*Async` version of `JsonSerializer`.

Reply:
Do you want it like this?

```csharp
using var fileStream = new FileStream(SentryOptionsJsonPathFull, FileMode.Create);
using var writer = new Utf8JsonWriter(fileStream);
writer.WriteStartObject();
writer.WriteString(nameof(unitySentryOptionsJson.Dsn), unitySentryOptionsJson.Dsn);
writer.WriteBoolean(nameof(unitySentryOptionsJson.Enabled), unitySentryOptionsJson.Enabled);
writer.WriteEndObject();
writer.Flush();
```

Reply:
yeah that's the approach on the .NET SDK, without reflection. A bit more work though